avm/src/daemon/daemon.rs

```diff
@@ -0,0 +1,28 @@
+use once_cell::sync::OnceCell;
+use tokio::task;
+use tokio::time::{Duration, sleep};
+
+use crate::vm::run::run;
```

Comment:
![run run](https://i.ytimg.com/vi/SqVtqbzTdVs/maxresdefault.jpg)

Reply:
lol do you want to leave `exec`? I actually changed the name

Reply:
Doesn't bother me either way. :)